src/deluge/storage/flash_storage.cpp

```diff
@@ -197,6 +202,9 @@ bool gridEmptyPadsCreateRec;
 bool gridAllowGreenSelection;
 GridDefaultActiveMode defaultGridActiveMode;
 
+
+enum defaultSliceMode;
```

Comment:
Not resolved @sichtbeton should be declared as SampleRepeatMode defaultSliceMode

Reply:
this is it